src/Sylius/Behat/Page/Admin/ShowToEditPageSwitcherInterface.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Behat\Page\Admin;
+
+use FriendsOfBehat\PageObjectExtension\Page\SymfonyPageInterface;
+
+interface ShowToEditPageSwitcherInterface extends SymfonyPageInterface
+{
+    public function hasEditPageButton(): bool;
+
+    public function switchToEditPage(): void;
+}
```

Comment:
This seems like overkill, furthermore, why do we have a specific interface for switching from show to edit, but not the other way?

Reply:
I split them because of the Creation pages. There is a scenario where it checks if the `show` button is not visible, but it will never transition from creation to the show page, therefore the `switchToShowPage` interface method would never be used.

In reference to `overkill` - would it be sufficient to create a single interface class, counterpart of `NavigationTrait` (or eventually, `NavigationElement`)?